### PR TITLE
Implemented Client Side Registration

### PR DESCRIPTION
**Pull request by Hristoki**
#72

### CODE REVIEW

**Review by YavorTsanev — APPROVED** [review]

File: UpSkill/ClientSide/Infrastructure/AuthenticationService.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public async Task<RegistrationResponseDto> RegisterUser(UserRegistrationDto inpu
         {
             var content = JsonSerializer.Serialize(input);
             var bodyContent = new StringContent(content, Encoding.UTF8, "application/json");
-            var registrationResult = await client.PostAsync("accounts/registration", bodyContent);
+            var registrationResult = await client.PostAsync("accounts/register", bodyContent);
             var registrationContent = await registrationResult.Content.ReadAsStringAsync();
```
Comment (NellyIfandieva) [inline UpSkill/ClientSide/Infrastructure/AuthenticationService.cs:27]: I aim to learn with this question: have been wondering: 2 rows down we check whether the registrationResult returned was _unsuccessful_. If yes, we quit the work of the method. 

So, why do we read the registrationContent, before we know, if the registrationResult was successful?

Comment (Hristoki) [reply UpSkill/ClientSide/Infrastructure/AuthenticationService.cs]: I see your point, I can't think of a reason why so I'll move it.

Comment (NellyIfandieva) [reply UpSkill/ClientSide/Infrastructure/AuthenticationService.cs]: ** :) **

**Review by NellyIfandieva — COMMENTED** [review]
(UPDATED below) We now have 2 Authentication services, at 2 diff places in the ClientSide project :) 

Will continue this comment, after I have a look at your Infrastructure folder in the same project (which I don't have) and make sure what makes more sense (from my POV only for the moment :) )

(UPDATE) Okay, your Infrastructure folder now contains IRegistrationService and RegistrationService. So, you have no AuthenticationService, and maybe that is why I made mine elsewhere. Should be no conflict, I hope, of any kind.

**Comment by Hristoki** [comment]
The current registration works only if api server is hosted on the 5001 port.

**Review by YavorTsanev — APPROVED** [review]

**Review by NellyIfandieva — APPROVED** [review]
So far so good.

**Pull request merged**